install.ps1

```diff
@@ -38,10 +39,15 @@ tar.exe xf $DenoZip -C $BinDir
 
 Remove-Item $DenoZip
 
-$User = [System.EnvironmentVariableTarget]::User
-$Path = [System.Environment]::GetEnvironmentVariable('Path', $User)
+$EnvTarget = if (Test-Path variable:\g) {
```

Comment:
Maybe instead of `$g` we should use something like `$denoGlobalInstall`/`$denoInstallGlobally` or something long and specific. That way it doesn't accidentally install globally for someone who happens to have `$g` variable and then they get confused. With `$v` it's not so bad if it accidentally happens.

Reply:
Okay, yes I agree.